apps/test/unit/templates/lessonOverview/StyledCodeBlockTest.jsx

```diff
@@ -58,4 +58,21 @@ describe('StyledCodeBlock', () => {
       '[`playSound`](/docs/applab/playSound)'
     );
   });
+
+  it('embeds block if blockName is provided', () => {
+    const wrapper = shallow(
+      <StyledCodeBlock
+        programmingExpression={{
+          syntax: 'playSound',
+          color: null,
+          link: '/docs/spritelab/playSound',
+          blockName: 'playSound'
+        }}
+      />
+    );
+
+    const blockLink = wrapper.find('a');
+    expect(blockLink.props().href).to.equal('/docs/spritelab/playSound');
+    expect(blockLink.find('div').length).to.equal(1);
```

Comment:
I am also confused about what this is testing. do these statments verify that blockly is being rendered on the page? if they do, it could be helpful to note how/why they imply this in the comments